.drone.yml

```diff
@@ -335,3 +335,54 @@ steps:
     instance:
       - drone-publish.rancher.io
 
+---
+kind: pipeline
+name: harvester-plugin
+
+depends_on:
+- test
+
+trigger:
+  ref:
+  - "refs/heads/release-harvester-plugin-v*"
+  - "refs/heads/master"
+  event:
+  - push
+  - tag
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: build
```

Comment:
Is this step needed?

Reply:
I'll verify it tomorrow.

Reply:
I think this `build` step is required, otherwise the following error will occur.

<img width="1605" alt="image" src="https://user-images.githubusercontent.com/24985926/189087847-3b7ab06f-d501-4c8a-a4a5-4574d7dbb9db.png">

Reply:
Ah, looks like the build-pkg script depends on that npm package. This could be resolved by replacing this step with 
```
cd $(dirname $0)/..
yarn --pure-lockfile install
```

That should avoid executing the rather long dashboard build process